test/Microsoft.AspNet.Mvc.Core.Test/OutputFormatterTests.cs

```diff
@@ -76,24 +76,31 @@ public static IEnumerable<object[]> SelectResponseCharacterEncodingData
         }
 
         [Fact]
-        public void WriteResponseContentHeaders_FormatterWithNoEncoding_Throws()
+        public void WriteResponseContentHeaders_NoSupportedEncodings_NoEncodingIsSet()
```

Comment:
Both of these scenarios are reasons why `SelectEncoding` might return `null`, which is what's covered here. 

I could add another test using mocks that does the same thing this one does. I don't really seem much value add.